spec/requests/blogs_spec.rb

```diff
@@ -87,7 +87,7 @@
       get "/blog/#{post1.posted_year}/#{post1.posted_month}"
 
       expect( response      ).to     have_http_status :ok
-      # TODO: expect( response.body ).to include 'February'
+      expect( response.body ).to     include 'February'
```

Comment:
Consider using the `have_css` matcher over the `include` matcher to more precisely target the expectation, and less prone to false positives.